src/domains/transaction/components/TransactionTable/PendingTransactionsTable/PendingTransactionsTable.test.tsx

```diff
@@ -697,6 +700,70 @@ describe("Signed Transaction Table", () => {
 		vi.restoreAllMocks();
 	});
 
+	it("should handle onClick on a mobile pending transfer row hen the transaction id is clicked", async () => {
+		const onClick = vi.fn();
+		mockPendingTransfers(wallet);
+
+		render(<PendingTransferRowMobile wallet={wallet} transaction={fixtures.transfer} onRowClick={onClick} />);
+
+		await userEvent.click(screen.getByTestId("PendingTransactionRow__transaction-id"));
+
+		expect(onClick).toHaveBeenCalledWith(fixtures.transfer);
+
+		vi.restoreAllMocks();
+	});
+
+	it("should render N/A when a signed transaction row has an unvalid timestamp in mobile", () => {
+		const onClick = vi.fn();
+		mockPendingTransfers(wallet);
+		vi.spyOn(fixtures.transfer, "timestamp").mockReturnValue(undefined);
+
+		render(
+			<SignedTransactionRowMobile
+				wallet={wallet}
+				transaction={fixtures.transfer}
+				onRowClick={onClick}
+				isCompact={true}
+			/>,
+		);
+
+		expect(screen.getAllByText("N/A")).toHaveLength(2);
+
+		vi.restoreAllMocks();
+	});
+
+	it("should render tooltip if the transaction cannot be deleted", async () => {
+		const onClick = vi.fn();
+		mockPendingTransfers(wallet);
+		vi.spyOn(fixtures.transfer?.wallet(), "publicKey").mockReturnValue("test");
+
+		render(<SignedTransactionRowMobile transaction={fixtures.transfer} wallet={wallet} onRowClick={onClick} />);
+
+		expect(
+			screen.queryByText(translations.TRANSACTION.MULTISIGNATURE.PARTICIPANTS_CAN_REMOVE_PENDING_MUSIG),
+		).not.toBeInTheDocument();
+
+		await userEvent.hover(screen.getAllByTestId("SignedTransactionRowMobile--remove")[0]);
+
+		expect(
+			screen.getByText(translations.TRANSACTION.MULTISIGNATURE.PARTICIPANTS_CAN_REMOVE_PENDING_MUSIG),
+		).toBeInTheDocument();
+
+		vi.restoreAllMocks();
+	});
+
+	it("should render N/A when a mobile pending transfer has an unvalid timestamp", () => {
+		const onClick = vi.fn();
```

Comment:
Note: We can apply this fix to the unit tests for the desktop table, too. Will do it tomorrow in a new PR.